design/pipeline-artifacts.md

```diff
@@ -131,16 +137,19 @@ jobs:
 If the `path` key is specified, it's a relative path from `$(Pipeline.Workspace)`.
 Directory names are not automatically injected by the pipeline anymore (but of course, directories present in the artifact itself are still used).
 
-If you include a `path`, `artifact` becomes required.
-Otherwise, we would risk sticking multiple artifacts' contents together in the same directory, clobbering files unpredictably.
+If you mention an `artifact` and also include a `path`, all the contents of the aritfact are downloaded to the path provided.
+However, if you include a `path` but don't mention any `artifact`, all the available artifacts are downloaded as sub folders with in the `path`. 
 
 Example:
 ```yaml
 jobs:
-- job: makeartifact
+- job: myjob
   steps:
   - script: ./build.sh
   - upload: outputs/**/*
+    artifact: WebTier
+  - upload: daata/**/*
+    artifact: DBTier
 
 - job: useartifact
   dependsOn: makeartifact
```

Comment:
`dependsOn` should be `myjob`. Additionally, it might be clearer to just call the first job `job1` and the second `job2`.

Reply:
Good catch. Updated it now.